CONTRIBUTING.md

```diff
@@ -39,6 +39,23 @@ See `git help commit`:
 
 Once changes have been merged, the release job will automatically run to package and release changed charts.
 
+### Create new chart
+```
+# Clone
+git clone
+cd charts
+sh -c "$(curl --location https://taskfile.dev/install.sh)" -- -d -b .bin
+
+# Create chart
```

Comment:
I'd add a note in here mentioning after this is generated some files require manual updating.

Reply:
Yes, maybe also show a quick message to the user after the creation is complete that `Chart.yaml` and `values.yaml` must be updated manually?